server/src/schema/schema.graphql

```diff
@@ -122,7 +122,12 @@ type OpenDocResult {
 
 type Mutation {
   ngAdd(path: String!, name: String!): CommandStarted
-  ngNew(path: String!, name: String!, collection: String!): CommandStarted
+  ngNew(
+    path: String!
+    name: String
+    collection: String!
+    newCommand: [String!]!
```

Comment:
doesn't have to be done in this PR, but it would be great to ! to other option lists in the schema.

Reply:
Totally!